resources/articles/platform_adhesion/raft_speed.md

```diff
@@ -1,7 +1,7 @@
 Raft Print Speed
 ====
-This setting determines the overall speed at which the raft is printed. The actual speed to print the raft at may still be different if the speed is specialised for the base, middle or top layers.
+This setting determines the overall speed at which the raft is printed. The actual speed at which the raft is printed may differ from this if specific speeds are set for the base, middle or top layers. 
 
 Printing the raft more slowly than the rest of the print is generally beneficial. Printing slower allows the plastic to anneal more because it stays hot for longer, which reduces the internal stresses within the lines and reduces warping. Because it stays hot for longer, the material can also flow out more, increasing the contact area with the bed which improves adhesion. In general, your raft will be stiffer, stronger, more reliable and stick better.
```

Comment:
```suggestion
Printing the raft more slowly than the rest of the print is generally beneficial. Printing slower allows the plastic to anneal more because it stays hot for longer, which reduces the internal stresses within the lines and reduces warping. Because it stays hot for longer, the material can also flow out more, increasing the contact area with the bed which improves adhesion. In general, your raft will be stiffer, stronger, more reliable and stick better. However, the raft will take more time to print.
```